tests/testGetInstcal.py

```diff
@@ -85,9 +85,6 @@ def testInstcal(self):
         self.assertAlmostEqual(weather.getAirPressure(), visit229388_info['weath_airPressure'])
         self.assertAlmostEqual(weather.getHumidity(), visit229388_info['weath_humidity'])
 
-        # Example of metadata which should *not* have been copied from zeroth extension.
-        self.assertNotIn("PROPOSER", exp.getMetadata().paramNames())
```

Comment:
Probably, but we also need to remove the special-case code someone added to obs_decam to work around this bug.  So I'll defer the testing to that ticket (DM-11264)